### ISSUE TITLE
Audio files dissapear after applying on item on Aternos servers

### ISSUE THREAD

**Issue by iCatSG** [henkelmax/audio-player#92]
### Bug description

When using it in an Aternos server, if you upload the mp3 or wav file directly to the `audplayer_uploads` folder (or use Filebin to upload it to a website so then gets downloaded onto the same folder) and then set it on any item, the file gets deleted or dissapears and only the ones that you try to apply to an item (so if you upload 10 mp3 files and only apply 1 o them, only that file that you apply gets deleted or dissapears from the foler and the rest of them stay).

I've tried the mod with the same version and same sound files on my private server and it works fine. When looking into my private server files folder, inside `audio_player_data`, the files are still there, just renamed to an specific ID. (I've uploaded some screenshots so you can see how it looks on my world and how it looks on Aternos).

I've also contacted the support team at Aternos and they told me that their servers don't have any mechanic to delete files so it should be a problem with the mod.

### Minecraft version

1.21

### Simple Voice Chat version

1.21.1 2.5.23

### Mod version

1.21.1 1.11.0

### Mod loader and version

Fabric Loader 0.16.5

### Steps to reproduce

1. Drag and drop the audio files onto the `audioplayer_uploads` folder in the Aternos website file manager.
2. While holding a disc type the command `/audioplayer serverfile "audiofilename.mp3" `.
3. Select in chat "Put on music disc" (while still holding the music disc).
4. Put the disc inside a jukebox.

### Expected behavior

No sound comes out of the jukebox even after triple checking sound options in the Minecraft Launcher and game.

### Log files

https://gist.github.com/iCatSG/08e1e14ea5a6c090f83f16ec8c05bf75

**Comment by henkelmax** [comment]
This is completely intended. Once the file is processed it gets deleted from the folder.

**Comment by iCatSG** [comment]
Hello again.
I think there was a misunderstanding.
The files get deleted, yeah but it leaves no files for the disc to play
since when I try to use the disc no sound plays and it's completly silence.
So in conclusion, after using all the commands needed and setting up the
disc, there no sound when putting insidr the jukebox

King regards,
iCatSG

**Comment by henkelmax** [comment]
this happens for every method of uploading a file?

**Comment by iCatSG** [comment]
Yes. I've tried all and I get the same result. No sound when using the disc
on the jukebox.

**Comment by henkelmax** [comment]
are there any files in the `audio_player_data` in your world directory?

**Comment by iCatSG** [comment]
When trying on the server, let's say I upload an audio file that I wouls
then use on the disc, after applying the audio to the disc and checking on
the folder there aren't any files.

But when trying on my own world, I upload the same file and the file I
uploaded gets deleted but there's another mp3 or wav that has the id as the
file name (which I believe is intended).

**Comment by iCatSG** [comment]
Also. I would like to mention that, in the Aternos server the folder it
created to upload the sounds is called "audioplayer_uploads" and in the
private world is called "audio_player_data".

**Comment by henkelmax** [comment]
You are confusing the two different folders. `audioplayer_uploads` is for you to insert files. the will get deleted and put into the system. `audio_player_data` is the folder you **SHOULD NOT TOUCH** this is where the files are stored with its assigned IDs

**Comment by iCatSG** [comment]
Okey got it but still, the disc won't play any sound. I've now checked to
fins the audio_player_data folder on the server and it has all the audio
files with the files' names been the ids but for some reason it doesn't
play the sound as intended

**Comment by iCatSG** [comment]
Do you have any specific steps to do so? I've set Simple Voice Chat on the
server and I can create groups and all, just haven't tested it with a
friend to see if I can hear my friend.

If you need me to test it with a friend I can do that this afternoon (I
live in Spain)

**Comment by henkelmax** [comment]
You need to get the voice chat working before using audioplayer.

**Comment by iCatSG** [comment]
Okey, I will do so now. I will let you know if it works

**Comment by iCatSG** [comment]
UPTADE: It works now. I thought the mod on the server side was setup
already since it didn't give any errors.

Sorry about this. It probably was pretty obvious that it was needed, I just
forgot to do it.
